lib/mongoose_push_web/controllers/api_v1_controller.ex

```diff
@@ -0,0 +1,34 @@
+defmodule MongoosePushWeb.APIv1Controller do
+  alias MongoosePushWeb.Schemas
+  alias OpenApiSpex.Operation
+  use MongoosePushWeb, :controller
+
+  @spec open_api_operation(atom) :: Operation.t()
+  def open_api_operation(action) do
```

Comment:
We're gonna have 3 APIs, 3 controllers, yet this function will always be the same. Also, the dynamic function apply is not "pretty", yet required, so hiding this would IMHO be helpful. How about injecting this function with `using` macro that could come with `MongoosePushWeb.Schemas` module? We would endup with:

```
use MongoosePushWeb.Schemas
```